### ISSUE TITLE
Backup zip does not restore "remote_playlist" table in newpipe.db file when it was existing in app databases folder

### ISSUE THREAD

**Issue by mbkasder** [TeamNewPipe/NewPipe#2508]
- [ x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [ x] I checked if the issue/feature exists in the latest version.
- [ ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.
Iam using newpipe version 0.17.0 
I have a playlist(pre existing one) not i created in bookmarked playlist section ,when I do backup of content in export database then later I wipe data and import the database I don't see the playlist(pre made by someone or channel itself) that i have added but the playlist i made can be visible.
So I open the databse to figure  out ,
I found that the table remote_playlist is empty in exported zip ,but it is healthy in /databases/newpipe.db
I am new user of github if I have done any mistake forgive me,and I hope I have brought this to developer notice.
Thank you for such  a amazing app :-)

**Comment by theScrabi** [comment]
I could not confirm this.
Things I tested:
Exporting database from 0.17.0, and Import it -> Playlists are there again
Exporting database from 0.16.4 and import it in 0.17.0 -> playlists are there again

1. Could you please tell us which android version you are using?
2. Would you have an issue sending us your backed up database? Though please modify it before you send it, so we cant see what videos you are watching.

**Comment by mbkasder** [comment]
This is the database exported from newpipe(I have deleted the subsciption table only):
[newpipe(exported from settings).zip](https://github.com/TeamNewPipe/NewPipe/files/3496605/newpipe.exported.from.settings.zip)

This is the database extracted from data/databases folder of newpipe(I have only deleted the subscription table):
[newpipe(extract from data).zip](https://github.com/TeamNewPipe/NewPipe/files/3496607/newpipe.extract.from.data.zip)

**Comment by mbkasder** [comment]
theScrabi ,
Did you confirmed it ?
What I did was opened a library or playlist in youtube search and clicked on add to playlist icon.
I observed that in newpipe.db the entry in remote_playlist is made which has link of playlist and icon info .
But when i did export option in app ,I noticed that same entry was missing in remote_playlist of newpipe.db in exported file . 
I had imported the backup before noticing the problem , Later I found this problem that I am creating issue that the table is not exported properly .
I am not sure its only happening in my device though.
Hoping to get solution.
